tsconfig.base.json

```diff
@@ -11,7 +11,7 @@
     // "disableReferencedProjectLoad": true,             /* Reduce the number of projects loaded automatically by TypeScript. */
 
     /* Language and Environment */
-    "target": "ES2021" /* Set the JavaScript language version for emitted JavaScript and include compatible library declarations. */,
+    "target": "ES2022" /* Set the JavaScript language version for emitted JavaScript and include compatible library declarations. */,
```

Comment:
how come? (it's fine, just checking why)

Reply:
Ran into an `top level await is not available` issue when building on my machine. The source of the issue was some package (don't remember exactly which) calling `await import(node:crypto)`